sentry/src/main/java/io/sentry/connection/HttpConnection.java

```diff
@@ -243,4 +282,4 @@ public void setBypassSecurity(boolean bypassSecurity) {
     @Override
     public void close() throws IOException {
     }
-}
+}
```

Comment:
Our checkstyle requires this newline: https://travis-ci.org/getsentry/sentry-java/jobs/353447874#L864